src/ring_theory/valuation/quotient.lean

```diff
@@ -0,0 +1,94 @@
+/-
+Copyright (c) 2020 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Buzzard, Johan Commelin, Patrick Massot
+-/
+
+import ring_theory.valuation.basic
+import ring_theory.ideal.quotient_operations
+
+/-!
+# The valuation on a quotient ring
+-/
+
+namespace valuation
+
+variables {R Γ₀ : Type*} [comm_ring R] [linear_ordered_comm_monoid_with_zero Γ₀]
+variables (v : valuation R Γ₀)
+
+/-- The extension of valuation v on R to valuation on R/J if J ⊆ supp v -/
+def on_quot {J : ideal R} (hJ : J ≤ supp v) :
```

Comment:
The docs for this are still left in the original file I think

Reply:
Good point! Moved.